Phonebook.Frontend/src/app/dialogs/dialog-view/dialog-view.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { DialogViewComponent } from './dialog-view.component';
+
+describe('DialogViewComponent', () => {
+  // let component: DialogViewComponent;
+  // let fixture: ComponentFixture<DialogViewComponent>;
+  // beforeEach(async(() => {
+  //   TestBed.configureTestingModule({
+  //     declarations: [DialogViewComponent],
+  //   }).compileComponents();
+  // }));
+  // beforeEach(() => {
+  //   fixture = TestBed.createComponent(DialogViewComponent);
+  //   component = fixture.componentInstance;
+  //   fixture.detectChanges();
+  // });
+  // it('should create', () => {
+  //   expect(component).toBeTruthy();
+  // });
```

Comment:
Why did you comment this out in all components?

Reply:
I‘m pretty new to Angular, and there were many errors when running the test script, caused by the spec file. I tried fixing them but ended up just commenting out the test section for now

Reply:
as long as there haven't been running tests before that's okay